frontend/src/actions/github/fetch_repos.js

```diff
@@ -1,16 +1,15 @@
-import { getThirdParty } from '../../lib/api';
+import { get } from '../../lib/api';
 import * as constants from '../../constants/github';
 
 
 /**
  * Fetch a user's repo
  */
 
-export default (username) => {
+export default (username, token) => {
   return dispatch => {
     dispatch(request(username));
-
-    return getThirdParty(`https://api.github.com/users/${username}/repos?per_page=100`, {params: {type: 'owner,member'}})
+    return get(`/../github/repositories/${token}`)
```

Comment:
That should work without needing a new method (since all the other calls to the API use the same function). 

For ex: https://github.com/OpenCollective/opencollective-website/blob/master/frontend/src/actions/groups/get_leaderboard.js